nerfstudio/pipelines/base_pipeline.py

```diff
@@ -183,12 +183,15 @@ def get_eval_image_metrics_and_images(self, step: int):
 
     @abstractmethod
     @profiler.time_function
-    def get_average_eval_image_metrics(self, step: Optional[int] = None, output_path: Optional[Path] = None):
+    def get_average_eval_image_metrics(
+        self, step: Optional[int] = None, output_path: Optional[Path] = None, get_std: Optional[bool] = None
```

Comment:
Any reason to use `Optional`?, why not `get_std: bool = False`

Reply:
No reason. I revised the default value to False.